packages/react-charts/src/components/Patterns/examples/patterms.md

```diff
@@ -0,0 +1,791 @@
+---
+id: Patterns
+section: charts
+propComponents: [
+  'ChartLegend'
+]
+hideDarkMode: true
+---
+
+import { 
+  Chart,
+  ChartArea,
+  ChartAxis,
+  ChartBar,
+  ChartDonut,
+  ChartDonutThreshold,
+  ChartDonutUtilization,
+  ChartGroup,
+  ChartLegend,
+  ChartLegendTooltip,
+  ChartPie,
+  ChartScatter,
+  ChartStack,
+  ChartThemeColor,
+  ChartVoronoiContainer,
+  createContainer,
+  getInteractiveLegendEvents,
+  getInteractiveLegendItemStyles,
+  getResizeObserver
+} from '@patternfly/react-charts';
+import chart_area_Opacity from '@patternfly/react-tokens/dist/esm/chart_area_Opacity';
+import chart_color_black_500 from '@patternfly/react-tokens/dist/esm/chart_color_black_500';
+import chart_color_blue_300 from '@patternfly/react-tokens/dist/esm/chart_color_blue_300';
+import chart_color_green_300 from '@patternfly/react-tokens/dist/esm/chart_color_green_300';
+import chart_color_cyan_300 from '@patternfly/react-tokens/dist/esm/chart_color_cyan_300';
+import chart_color_gold_300 from '@patternfly/react-tokens/dist/esm/chart_color_gold_300';
+import '@patternfly/patternfly/patternfly-charts.css';
+
+## Introduction
+Note: PatternFly React charts live in its own package at [@patternfly/react-charts](https://www.npmjs.com/package/@patternfly/react-charts)!
+
+PatternFly React charts are based on the [Victory](https://formidable.com/open-source/victory/docs/victory-chart/) chart library, along with additional functionality, custom components, and theming for PatternFly. This provides a collection of React based components you can use to build PatternFly patterns with consistent markup, styling, and behavior.
+
+## Examples
+### Basic pie chart
+```js
+import React from 'react';
+import { ChartPie, ChartThemeColor } from '@patternfly/react-charts';
```

Comment:
ChartThemeColor is not used in this example

Reply:
Done